### ISSUE TITLE
Panel elements still clickable even if not visible

### ISSUE THREAD

**Issue by spcified** [fthx/hide-panel-lite#2]
While the panel is hidden the elements can still be accessed. Also animations like spinners are still drawn even if no panel is available.

![image](https://user-images.githubusercontent.com/9081713/178788225-f4eabaad-f00f-40c1-b941-8df65d03ed40.png)

Did not test, but would a negative position be possible instead of 0 height?

**Comment by ghost** [comment]
Confirm the same here. But I kinda like this. It works as hot corner shortcuts for me without panel being visible. It's kinda usuable & also productive too.

**Comment by spcified** [comment]
@pc00per, yes, this is only annoying if you have tabs ob top without any title bar.

**Comment by fthx** [comment]
Hi all,

I'm open to some PR but it has to keep it very simple (that's the goal of this extension).
ATM I'm on https://extensions.gnome.org/extension/5143/dashbar/ so I do use the top panel actually!
I really recommend this new extension, so much lighter than dash to panel (you just get the dash in panel, taken from GS code itself).

**Comment by spcified** [comment]
https://github.com/mlutfy/hidetopbar does the trick almost all of the time but it es everything but light. From the looks it the other extensions uses a negative height and calls additionally PanelBox.hide().

@fthx do you know if there are any docs regarding the PanelBox functions? My search results only lead me to issues regarding other extensions.
